########################################
# Audio frame in frequency domain (squared magnitude)
########################################

Header header

uint32 frameLength
uint32 nChannels
uint32 fs
float32[] data